shell.nix

```diff
@@ -8,34 +8,20 @@ let
     overlays = [ ];
   };
 
-  bky-as = pkgs.buildGoModule rec {
+  bky-as = pkgs.stdenv.mkDerivation rec {
     pname = "bky-as";
     version = "v0.1.0-beta.4";
 
-    tmpDir = "/tmp";
-
-    env = {
-      GOPRIVATE = "github.com/blocky/*";
-      HOME = tmpDir;
-    };
-
-    src = builtins.fetchGit {
-      ref = version;
-      url = "git@github.com:blocky/delphi.git";
+    src = pkgs.fetchurl {
+      url = "https://github.com/blocky/attestation-service-demo/releases/download/v0.1.0-beta.4/bky-as_linux_amd64";
+      sha256 = "sha256-Gm/zEiP1jJaloxN6TdiCq112zk6hHlTWu65VkUlNceU=";
```

Comment:
where do these shas come from? did you compute this?

Reply:
You can either get it using `nix-prefetch-url` with:
```
nix-prefetch-url https://github.com/blocky/attestation-service-demo/releases/download/v0.1.0-beta.4/bky-as_linux_amd64
```

Alternatively, people sometime will just write it as:
```
  src = pkgs.fetchurl {
      url = "https://github.com/blocky/attestation-service-demo/releases/download/v0.1.0-beta.4/bky-as_linux_amd64";
      sha256 = "sha256-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=";
```

run `nix-build` and the "correct" sha will be in the output of the build failure.